helm/cluster-openstack/templates/coredns.yaml

```diff
@@ -0,0 +1 @@
+{{- include "coredns" .}}
```

Comment:
I'd add a comment this comes from `cluster-shared` because it's really difficult to realise if you ask me.